tests/utils.go

```diff
@@ -706,3 +707,38 @@ func createNodeDaemonSet(ds *appsv1.DaemonSet) {
 		gomega.BeNil(),
 		"creating node plugin daemonset %v", nodeDaemonSet)
 }
+
+func enableThinpoolMonitoring() {
+	lv := VOLGROUP + "/" + pvcObj.Spec.VolumeName
+	vol, err := lvm.ListLVMLogicalVolume(lv)
```

Comment:
For getting thinpool lv using lvs. Probably we can write a separate function. As the existing `ListLVMLogicalVolume` uses `lvs_all` in --options. This returns so much addition info related to stats which we might not need for this?

Reply:
I've re-written now to reuse the existing `execAtLocal` utility function, and left `ListLVMLogicalVolume` untouched.